docs/source/conservedwaters.rst

```diff
@@ -1,8 +1,35 @@
-Theory, Background and Methods
-==============================
-
-What is a conserved water?
-What types of conserved waters exist?
-Where can we use this?
-Classification of conserved waters.
-See `here <https://doi.org/10.1021/acs.jcim.2c00801>`_.
\ No newline at end of file
+Theory, Background, and Methods
+===============================
+
+The developed method focuses on the identification and classification of conserved water molecules and their networks derived from molecular dynamics (MD) simulations. A unique feature of this method is its ability to not only determine the positions of oxygen atoms but also the orientations of hydrogen atoms. This dual focus enables a nuanced classification into types like FCW, HCW, and WCW, which is a significant advantage over existing methods. The classification provides vital information for assessing the water network's stability. The method operates solely on data from MD simulations, without requiring any crystal water data.
```

Comment:
FCW, HCW, WCW have not been defined yet? Maybe we could have a glossary at the beginning of the page? Or you could state that the upside of the approach is the possibility to classify them as WCW , FCW and HCW with brief explanations on what makes them different?

Reply:
I would move this whole section after "understanding conserved waters" section.